qt/scientific_interfaces/MultiDatasetFit/MultiDatasetFit.cpp

```diff
@@ -389,6 +384,10 @@ void MultiDatasetFit::fit() {
 
   auto fittingType = m_fitOptionsBrowser->getCurrentFittingType();
   auto n = getNumberOfSpectra();
+  if (n == 0) {
+    QMessageBox::warning(this, "MantidPlot - Warning", "Data wasn't set.");
```

Comment:
Is there a way we could make this warning not hardcoded to display MantidPlot, so that it will display workbench if provoked from the workbench for example.

Reply:
I can change it to Mantid - Warning